fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalTopN.java

```diff
@@ -39,7 +39,10 @@
  * Physical top-N plan.
  */
 public class PhysicalTopN<CHILD_TYPE extends Plan> extends AbstractPhysicalSort<CHILD_TYPE> implements TopN {
-    public static String TOPN_RUNTIME_FILTER = "topn_runtime_filter";
+
+    public static final String TOPN_OPT = "topn_opt";
```

Comment:
why rename TOPN_RUNTIME_FILTER to TOPN_OPT?
both topn-runtime-filter and topn-two-phase-read are opt of topN.

Reply:
change back in next pr